### PR TITLE
Improve header and footer

### PR DESCRIPTION
**Pull request by user512**
Connects: https://github.com/zinc-collective/www.zinc.coop/issues/13
Screen Shot 2020-05-26 at 5 25 07 PM
Screen Shot 2020-05-26 at 5 25 00 PM

Mobile isn't optimized yet but will work on it in a follow up PR.
Screen Shot 2020-05-26 at 5 26 43 PM

### CODE REVIEW

**Comment by maximegalon5** [comment]
This looks great! Thanks.

File: src/_includes/partials/footer.njk [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-    </ul>
-
-    <ul  class="w-full sm:w-full md:w-1/2 lg:w-1/2 xl:w-1/2">
-      <li><a href="/operating-agreement">Operating Agreement</a></li>
-      <li><a href="/code-of-conduct">Code of Conduct</a></li>
-      <li><a href="/financial-overview">Financial Overview</a></li>
-      <li><a href="/meeting-notes">Meeting Notes</a></li>
-      <li><a href="/project-guidelines">Project Guidelines</a></li>
-    </ul>
-  </nav>
```
Comment (zspencer) [inline src/_includes/partials/footer.njk:18]: I strongly believe we want to leave some of this in the footer. Footers are a great place to put "signposts" to useful-but-not-core trails of information.

When we put that kind of thing behind drop-downs; we make it harder for folks who use screen readers to navigate the website.

Could you reach out to @colombene to get her thoughts on how to organize website navigation?

File: src/product-and-services/index.md [hunk 2]
```diff
@@ -4,20 +4,24 @@ title: Products and Services
 ---
 ## In Active Development
 
-### [Notabli](https://www.notabli.com)
+### [Convene](https://www.zinc.coop/convene)
 
-Notabli helps parents safely share their children's childhood moments with their extended family. We spend four to eight days a month updating [Notabli](https://www.notabli.com).
+[Convene](https://www.zinc.coop/convene)(Convene creates human connection through safe, accessible and reliable video conferencing.
```
Comment (zspencer) [inline src/product-and-services/index.md:9]: `/www.zinc.coop/convene)(Convene creates human`

It looks like there's an extra`(` here in `zinc.coop/convene)(Convene`

**Review by zspencer — COMMENTED** [review]
I think this is good; but I think this is the second or third time we've moved the footer navigation around. I would appreciate it if we took the time to figure out _where_ we want our tertiary links to live; and then figure out a way to communicate that reasoning in with where the links wind up; so that it can be re-evaluated later but it stops creating churn.

**Review by maximegalon5 — COMMENTED** [review]
I agree with @zspencer that those using screen readers will have a tougher time navigating drop-down menus.

File: src/_includes/partials/footer.njk (hunk 1 above)
Comment (user512) [reply src/_includes/partials/footer.njk:18]: Yes, I was under the impression from the last design session that we would like to consolidate them into header and group them under Join us or About us according to the site map but I didn't confirm. I can reach out and get thoughts on this.

File: src/product-and-services/index.md (hunk 2 above)
Comment (user512) [reply src/product-and-services/index.md:9]: Good catch. Fixed in https://github.com/zinc-collective/www.zinc.coop/pull/51

**Comment by user512** [comment]
Good call on screen readers users. 
I made some changes in https://github.com/zinc-collective/www.zinc.coop/pull/51 that should resolve screen readers user concerns. 

Also will reach out to get @colombene thoughts on a better way to present nav item.

**Pull request merged**